frontends/precinct-scanner/src/screens/poll_worker_screen.tsx

```diff
@@ -118,16 +121,23 @@ export function PollWorkerScreen({
   const hasPrinterAttached = printerFromProps || !window.kiosk;
   const { election } = electionDefinition;
 
-  const precinct = election.precincts.find((p) => p.id === currentPrecinctId);
+  const precinct =
+    currentPrecinctId === ALL_PRECINCTS_OPTION_VALUE
+      ? ALL_PRECINCTS_OPTION_VALUE
+      : election.precincts.find((p) => p.id === currentPrecinctId);
```

Comment:
You can use the `find` function from @votingworks/utils to automatically assert that the precinct is found

Reply:
Cool, thanks!